webknossos/webknossos/dataset/dataset.py

```diff
@@ -987,7 +989,10 @@ def add_layer_from_images(
         use_bioformats: bool = False,
         channel: Optional[int] = None,
         timepoint: Optional[int] = None,
+        czi_channel: Optional[int] = None,
```

Comment:
I have to admit that this whole method is growing quite large. It might be a good idea to extract it into its own module, but not quite sure.